student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person' # Require the person.rb file
+
+class Student < Person
+  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly look at the feedback correctly, you need to fix the `super` parameters as well since `age` is no longer optional to avoid errors:

  ```
  super(age, name: name, parent_permission: parent_permission)
  ```